src/client/components/RegisterTeaser/RegisterTeaser.css

```diff
@@ -0,0 +1,20 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@500&display=swap');
```

Comment:
You can now delete the import as we will use from the downloaded fonts instead :)